client/src/app/site/pages/meetings/pages/participants/export/participant-csv-export.service/participant-csv-export.service.ts

```diff
@@ -40,12 +43,16 @@ export class ParticipantCsvExportService {
     //     ]
     // ]);
 
+    public columns = Object.keys(participantHeadersAndVerboseNames).sort(
+        (a, b) => participantColumnsWeight[a] - participantColumnsWeight[b]
+    );
```

Comment:
If `participantColumnsWeight` already contains all relevant fields, why even import `participantHeadersAndVerboseNames` at all

Reply:
yes, changed it.